ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/ProcessableBlockHeader.java

```diff
@@ -46,6 +47,8 @@ public class ProcessableBlockHeader
   protected final Bytes32 mixHashOrPrevRandao;
   // parentBeaconBlockRoot is included for Cancun
   protected final Bytes32 parentBeaconBlockRoot;
+  // TODO SLD Quantity or UInt64Value<UInt64> instead?
+  protected final UInt64 targetBlobCount;
```

Comment:
Some other code for similar types use a subclass of BaseUInt256Value (which implements Quantity and UInt256Value). 
Withdrawals already uses UInt64 directly.

I think UInt64 is more direct than the tuweni generic base class construct and avoids yet another wrapper class.

Would like to get more opinions about the benefits of defining our own type, i.e. `TargetBlobCount extends BaseUInt256Value<TargetBlobCount> implements Quantity`

Reply:
Side note: Teku using UInt64 directly, as in this PR https://github.com/Consensys/teku/blob/110ce917f1fec942e2c2664c50a1bc3bb778ee07/ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/schema/PayloadAttributesV4.java#L48

Reply:
Fine with using UInt64 directly. Adding the new type doesn't add much value